group_vars/all.yml.sample

```diff
@@ -194,7 +198,11 @@ dummy:
 #ceph_dev_branch: master # development branch you would like to use e.g: master, wip-hack
 #ceph_dev_sha1: latest # distinct sha1 to use, defaults to 'latest' (as in latest built)
 
+#nfs_ganesha_dev: false # use development repos for nfs-ganesha
```

Comment:
you must edit ceph-common/defaults/main.yml and then run generate_group_vars_sample.sh